app/models/lend_request_notification.rb

```diff
@@ -0,0 +1,15 @@
+# Class specifically for lending request notifications
+class LendRequestNotification < ApplicationRecord
+  acts_as :notification
+
+  belongs_to :borrower, class_name: "User"
+  belongs_to :item
+
+  def title
+    I18n.t "views.notifications.lend_request.title"
+  end
+
+  def description
+    I18n.t "views.notifications.lend_request.description", user: borrower.name, item: item.name
+  end
+end
```

Comment:
Is this lend request notification included on purpose? As far as I'm aware, this is part of another user story

Reply:
The corresponding story was merged, so these changes are not included any more